build-tools/packages/version-tools/src/commands/version.ts

```diff
@@ -9,13 +9,15 @@ import { table } from "table";
 import { isVersionBumpTypeExtended } from "../bumpTypes";
 import {
     bumpInternalVersion,
-    DEFAULT_PUBLIC_VERSION,
     fromInternalScheme,
     toInternalScheme,
     getVersionRange,
     isInternalVersionScheme,
 } from "../internalVersionScheme";
 
+/** The default public version used to generate Fluid internal versions when none is provided. */
+const DEFAULT_PUBLIC_VERSION = "2.0.0";
```

Comment:
What is the long-term plan for this default? I'm always nervous when I see something like this codified.

Reply:
Good question. I figured that since it's a default it would remain "for all time". But I could just remove the default. It makes the command mildly more annoying to use, but it's probably the right tradeoff.